pycbc/inference/models/relbin.py

```diff
@@ -303,12 +292,12 @@ def __init__(
             self.fedges[ifo] = self.f[ifo][fbin_ind]
             self.edges[ifo] = fbin_ind
 
-            self.init_from_frequencies(h00, fbin_ind, ifo)
+            self.init_from_frequencies(data, self.h00, fbin_ind, ifo)
```

Comment:
Why are we sending `self.h00` as a variable to a class method? Why not just call `self.h00` in `init_from_frequencies`?